addons/account/models/account_tax.py

```diff
@@ -125,8 +122,12 @@ def _default_tax_group(self):
         default=True,
         help="If set, taxes with a lower sequence might affect this one, provided they try to do it.")
     analytic = fields.Boolean(string="Include in Analytic Cost", help="If set, the amount computed by this tax will be assigned to the same analytic account as the invoice line (if any)")
-    tax_group_id = fields.Many2one('account.tax.group', string="Tax Group", default=_default_tax_group, required=True,
-                                   domain="[('country_id', 'in', (country_id, False))]")
+    tax_group_id = fields.Many2one(
+        comodel_name='account.tax.group',
+        string="Tax Group",
+        compute='_compute_tax_group_id', readonly=False, store=True,
+        required=True, precompute=True,
```

Comment:
Because it is `required=True` (and so it is in the database)
If it wasn't precomputed, the ORM would try to insert before computing (so with a NULL required value), then compute; which would fail.